### PR TITLE
Add option for setting PID

### PR DESCRIPTION
**Pull request by syyyr**
Hi, now that there options for separating the sysrepo repository and shm prefix via environmental variables, the only thing that disallows launching more netopeer2 processes is the pidfile. This PR adds on option to set a custom pid file.

### CODE REVIEW

**Review by jktjkt — APPROVED** [review]
And just to clarify, the reason for running several `netopeer2-server` processes in the first place is automated unit tests of NETCONF-talking software -- the [`netconf-cli`](https://github.com/CESNET/netconf-cli) in our case. It is very convenient just to launch `netopeer2-server` and react to sysrepo callbacks from the test suite.

File: src/main.c [hunk 1]
```diff
@@ -1076,6 +1077,7 @@ main(int argc, char *argv[])
     int c, *idx, i;
     int daemonize = 1, verb = 0;
     int pidfd;
+    const char* pidfile = NP2SRV_PID_FILE_PATH;
```
Comment (michalvasko) [inline src/main.c:1080]: Minor formatting problem, use `char *` please.

**Review by michalvasko — CHANGES_REQUESTED** [review]
Generally seems fine, thanks.

File: src/main.c (hunk 1 above)
Comment (syyyr) [reply src/main.c]: done

**Review by michalvasko — APPROVED** [review]

**Pull request merged**